jobspec2/parse_test.go

```diff
@@ -10,14 +10,12 @@ import (
 	"time"
 
 	"github.com/hashicorp/nomad/api"
-	"github.com/hashicorp/nomad/ci"
-	"github.com/hashicorp/nomad/helper/pointer"
 	"github.com/shoenig/test/must"
 	"github.com/stretchr/testify/require"
 )
 
 func TestParse_ConnectJob(t *testing.T) {
-	ci.Parallel(t)
+	t.Parallel()
```

Comment:
if for some reason we wanted to retain `ci` here, it's a very short method, we could write it again here.

but it's easy to believe that these are fine to run in parallel, and I suspect probably quite a few of our tests could get un-`ci`'d like this and speed up test runs.

Reply:
Agreed.

I did a spot check on these tests and they look fine for parallel and the runtime looks shorter as a result of the change too. That being said, I'll keep an eye on them once merged.